backend/routes/food.js

```diff
@@ -4,5 +4,8 @@ const foodServices = require('../services/foods');
 const router = express.Router();
 
 router.post('/create', foodServices.createFood);
+router.get('/menu/:id', foodServices.getMenu);
+router.post('/update', foodServices.updateItem);
```

Comment:
The http request method for this should be PATCH because it is an update function. Also, the endpoint should be PATCH /:id, then read the food id from req.params